suite-native/message-system/src/components/MessageSystemRenderer.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactNode } from 'react';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import { useSelector } from 'react-redux';
+
+import { Box, VStack } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { ContextMessage } from './ContextMessage';
+import { selectAllActiveMessageIds } from '../selectors';
+
+type MessageSystemRendererProps = {
+    children: ReactNode;
+};
+
+const messageSystemRendererStyle = prepareNativeStyle<{ topSafeAreaInset: number }>(
+    (utils, { topSafeAreaInset }) => ({
+        position: 'absolute',
+        flex: 1,
+        flexDirection: 'row',
+        marginTop: topSafeAreaInset,
+        paddingHorizontal: utils.spacings.medium,
+    }),
+);
+
+export const MessageSystemRenderer = ({ children }: MessageSystemRendererProps) => {
+    const { applyStyle } = useNativeStyles();
+    const { top: topSafeAreaInset } = useSafeAreaInsets();
+    const activeMessagesIds = useSelector(selectAllActiveMessageIds);
+
+    return (
+        <>
+            {children}
```

Comment:
What is the benefit of wrapping children inside MessageSystemRenderer now? I can imagine some logic for not rendering children at all in case something is in the message, but nothing like that is implemented here, right?

Reply:
There is probably no benefit to it. We have like all the other renderers, so I sticked to this style to make it consistent across the app. 

But we will have to refactor all the renderers anyway when implementing notifications. We will revisit this in another issue, so please ignore it for now.